docs/tutorials/using_macros.md

```diff
@@ -0,0 +1,57 @@
+---
+icon: octicons/gear-24
+---
+
+# ⚙️ **Using Macros**
+
+Macros are a global set of hard-coded, "magic" numbers that are used as default values across **OmniGibson**. These values can have significant implications that broadly impact **OmniGibson**'s runtime (such as setting `HEADLESS` or `DEFAULT_PHYSICS_FREQ`), or can have a much more narrow scope that impacts only a specific module within **OmniGibson** (such as `FIRE_EMITTER_HEIGHT_RATIO`).
+
+All macros within **OmniGibson** can be directly accessed and set via the [`omnigibson/macros.py`](../reference/macros.html) module. There are two sets of macros:
+
+1. **Global Macros**: Accessed via the `gm` module variable, these are fundamental settings that generally impact all parts of **OmniGibson** runtime, and include values such as `gm.HEADLESS`, `gm.DEFAULT_PHYSICS_FREQ`, and `gm.ENABLE_HQ_RENDERING`.
```

Comment:
Do we have any documentation on what these global macros do?

Reply:
Yes! They're directly documented in the macros file itself. I will add a comment about that.